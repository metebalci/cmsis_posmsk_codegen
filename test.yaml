REG:
  - RESERVED: 2
  - FIELD_1: 3
  - RESERVED: 2
  - FIELD_2: 1
  - RESERVED: 24
