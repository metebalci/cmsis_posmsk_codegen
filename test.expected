#define TEST_REG_FIELD_1_Pos 27U
#define TEST_REG_FIELD_1_Msk (7UL << TEST_REG_FIELD_1_Pos)
#define TEST_REG_FIELD_2_Pos 24U
#define TEST_REG_FIELD_2_Msk (1UL << TEST_REG_FIELD_2_Pos)
